app/Http/Controllers/Api/V1/PaletteController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Api\StorePaletteRequest;
+use App\Http\Requests\Api\UpdatePaletteRequest;
+use App\Http\Resources\PaletteIndexResource;
+use App\Http\Resources\StorePaletteResource;
+use App\Models\Palette;
+use App\Services\ApiResponse\ApiResponseFacade;
+use App\Services\Palette\PaletteService;
+use Illuminate\Support\Facades\Gate;
+
+class PaletteController extends Controller
+{
+    public function __construct(private PaletteService $paletteService)
+    {
+    }
+
+    public function index()
+    {
+        $palettes = Palette::with('user')->paginate(10);
+        return ApiResponseFacade::withSuccess(true)
+            ->withStatus(200)
```

Comment:
`200` status code is default behavior of this `ApiResponseFacade` so we don't need to set it again